cmd/snap/cmd_snapshot_test.go

```diff
@@ -64,8 +65,23 @@ var snapshotsTests = []getCmdArgs{{
 }, {
 	args:   "check-snapshot 4 snap1 snap2",
 	stdout: "Snapshot #4 of snaps \"snap1\", \"snap2\" verified successfully.\n",
+}, {
+	args:   "export-snapshot 1 snapshot-export.tar",
+	stdout: "Exported snapshot into \"snapshot-export.tar\"\n",
```

Comment:
it would be nice if we verified that the snapshot-export.tar has the expected contents (which IIUC would be `Hello world!`?)

Reply:
This is done now, thanks for the suggestion (and patch)